homeassistant/components/husqvarna_automower_ble/config_flow.py

```diff
@@ -34,8 +34,12 @@ def _is_supported(discovery_info: BluetoothServiceInfo):
         service == "98bd0001-0b0e-421a-84e5-ddbf75dc6de4"
         for service in discovery_info.service_uuids
     )
+    service_gardena = any(
+        service == "0000fd69-0000-1000-8000-00805f9b34fb"
+        for service in discovery_info.service_uuids
+    )
 
-    return manufacturer and service_husqvarna
+    return manufacturer and service_husqvarna and not service_gardena
```

Comment:
This is not guaranteed to exists as far as i know?

Reply:
Ie. We dont use that service for any valves or sprinklers. So i have no idea if it would exist on all gardena devices. What is this service?

Reply:
> Please just parse the manufacturer data. We know the format.

What is different between the two though?

Reply:
Between which two?

Current solution wont work since the valves and timers dont expose the service you are using as a filter here.